app/src/ui/changes/changes-list.tsx

```diff
@@ -377,6 +379,31 @@ export class ChangesList extends React.Component<
     showContextualMenu(items)
   }
 
+  private getNamesOfSelectedOversizedFiles = async () => {
+    const fileNames: string[] = []
+    const workingDirectoryFiles = this.props.workingDirectory.files
+
+    for (const file of workingDirectoryFiles) {
+      const fileIsSelected =
```

Comment:
:art: rather than doing this check in the for loop, you could filter the array up front:

```ts
const workingDirectoryFiles = this.props.workingDirectory.files

const selectedFiles = workingDirectoryFiles.filter(
  f => f.selection.getSelectionType() !== DiffSelectionType.None
)

for (const file of selectedFiles) {
  // ...
}
```